components/FlyForm.js

```diff
@@ -37,11 +37,6 @@ class FlyForm extends Component {
     this.props.updateFlyEntry('amount', text)
   }
 
-  checkState = (name, color, size, type, amount) => {
```

Comment:
Thanks for removing this and cleaning up the code now that we feel confident about what's happening and can do so.